docs/research.md

```diff
@@ -0,0 +1,36 @@
+<h1> How has research been involved in informing the weather.gov 2.0 design? </h1>
```

Comment:
For future reference. You don't need to use html markup for headings.

[hash symbols](https://www.markdownguide.org/basic-syntax/#headings) can be used instead